cdp_scrapers/instances/kingcounty-static.json

```diff
@@ -0,0 +1,166 @@
+{
+    "persons": {
+        "Rod Dembowski": {
+            "name": "Rod Dembowski",
+            "is_active": true,
+            "router_string": null,
+            "email": "rod.dembowski@kingcounty.gov",
+            "phone": "206-477-1001",
+            "website": "https://kingcounty.gov/council/dembowski.aspx",
+            "picture_uri": "https://kingcounty.gov/~/media/council/images/2018/Dembowski_2x3_print.ashx?la=en",
```

Comment:
I _think_ this image should work, but not entirely sure because its an "ashx" file.

Regardless, I tested it and it works just fine if you remove the `?la=en`, can you do that?

Reply:
I thought it was weird too. It's been WAY too long since I even looked at ASP.NET (guessing ashx is from it) but yeah looks like it just resolves to an image. Will remove the trailing query from all of them.